js-modern-syntax/js-modern-syntax_rest-and-spread/index.js

```diff
@@ -49,3 +52,5 @@ export const result = add(numbers);
 
 const fruits = ["apple", "banana", "orange", "papaya"];
 const anotherFruit = "cherry";
+const allFruits = [anotherFruit, ...fruits];
```

Comment:
Instead of ```
const allFruits = [anotherFruit, ...fruits];
export { allFruits };
```
I think it can be rewritten as following:
`export const allFruits = [anotherFruit, ...fruits]; `